src/libraries/System.Net.Http/tests/StressTests/HttpStress/windows.Dockerfile

```diff
@@ -14,6 +14,11 @@ COPY . .
 ARG CONFIGURATION=Release
 RUN dotnet build -c $env:CONFIGURATION
 
+# Enable dump collection
+ENV COMPlus_DbgEnableMiniDump=1
+ENV COMPlus_DbgMiniDumpType=MiniDumpWithFullMemory
+ENV COMPlus_DbgMiniDumpName="C:/share/coredump.%p"
```

Comment:
should it use the value in IMAGE_SHARE instead of "C:/share"? (same for linux dockerfile)

Reply:
The IMAGE_SHARE is variable only because Windows/Linux incompatibility. I'd have used a constant, but I cannot unless I duplicate the docker-compose definition file docker-compose.yml. It shouldn't be overridable or configurable.

If I wanted to use a variable here, I'd have to pass it in as ARG (see HTTPSTRESS_ARGS occurrences), at this moment it's not available in the image.

Reply:
Ha, I like @antonfirsov approach of making the volume definition string one env var: `export HTTPSTRESS_REPORT_VOLUME_MOUNT_STRING="$(Build.StagingDirectory):/logs"`
That makes more sense, I might change to that.

Reply:
Yeah I hated the idea of creating two env vars, though this is a bit more tricky stuff that has to be documented in comments.

Reply:
I'm keeping it as-is, we might consider consolidation after @antonfirsov change.